packages/components/src/common/icons.ts

```diff
@@ -31,4 +33,5 @@ export const SUPPORTED_ICONS_MAP: Record<SupportedIconName, SupportedIconType> =
     "office-building": BiBuildings,
     stars: BiStar,
     globe: BiGlobe,
+    minus: BiMinus,
```

Comment:
question: hmm the icon here means that we are making it available for users to use this icon (for instance in infocols), is this intended?

Reply:
removed in https://github.com/opengovsg/isomer/pull/377/commits/b6a81d55cccf307d154baf1d4ed64fa390f22e95